api/api.yaml

```diff
@@ -0,0 +1,17 @@
+swagger: '2.0'
+info:
+  title: Data Explorer Service
+  description: API Service that reads from Elasticsearch.
+  license:
+    name: BSD
+    url: 'http://opensource.org/licenses/BSD-3-Clause'
+  version: '0.0.1'
+paths:
+  /facets:
+    get:
+      description: Returns facets.
+      responses:
+        200:
+          description: Facets.
+          schema:
```

Comment:
I think this should return an array of facet strings instead of a single string?

Reply:
Will handle in future PRs. I'll create a Facets object which has list of Facet, etc.